embedded-graphics/src/primitives/circle.rs

```diff
@@ -272,6 +271,23 @@ mod tests {
     use super::*;
     use crate::pixelcolor::BinaryColor;
 
+    #[test]
+    fn issue_143_stroke_and_fill() {
```

Comment:
We should define a default scheme to refer to GitHub issues in tests. I don't care what it is but having a defined way would make it easier to grep for in the future. I think currently only one other test exists that refers to an issue:
https://github.com/jamwaffles/embedded-graphics/blob/9196c227f4acd8b1d52bd270fbe2cd9c6257ba98/embedded-graphics/src/image/image_bmp.rs#L340-L342
Could you change one or the other to match?

Reply:
Good shout. I've made the names and comments consistent in 51d3eb9.